UnityProject/Assets/Scripts/UI/Systems/MainHUD/UI Bottom/ControlAction.cs

```diff
@@ -12,13 +12,40 @@ public class ControlAction : MonoBehaviour
 
 		public Image pullImage;
 
+		public static bool ThrowHold = true;
+
 		private void Start()
 		{
 			UIManager.IsThrow = false;
 
 			pullImage.enabled = false;
+			ThrowHold = GetThrowPreference();
+		}
+
+		public static bool GetThrowPreference()
```

Comment:
naming this ``GetTrowHoldPreference`` makes it clearer than a ``true`` value means the throw behaviour is set to hold.